Kickstarter.xcodeproj/project.pbxproj

```diff
@@ -6497,6 +6525,7 @@
 					"@executable_path/Frameworks",
 					"@loader_path/Frameworks",
 				);
+				OTHER_LDFLAGS = "-ObjC";
```

Comment:
Not sure if we need this in the linker flags? At least I didn't during testing.

Reply:
Ah I was reading the directions here: https://firebase.google.com/docs/ios/setup#integrate-without-cocoapods and assumed those directions were relevant. Reading it again, I think that is only for situations where we are manually downloading the framework and dropping it into our project.

Reply:
Ah actually, I just ran a quick test and removed the linker flag and a Facebook framework throws an `unrecognized selector` error that is a result of the deletion of the linker flag. Going to leave this in.